cypress/udata-front-e2e.cfg.example

```diff
@@ -0,0 +1,10 @@
+PLUGINS = ['front']
+THEME = 'gouvfr'
+
+MONGODB_HOST = 'mongodb://localhost:27017/udata-front-e2e'
+POST_DISCUSSIONS_ENABLED = True
+
+CAPTCHETAT_CLIENT_ID =
```

Comment:
Maybe adding the CI here would make us think about the way to set these? I've done something similar for ligthhouse in .circleci/config.yml